joinmarket/wallet.py

```diff
@@ -143,7 +143,7 @@ def __init__(self,
         for i in range(self.max_mix_depth):
             self.index.append([0, 0])
 
-    def read_wallet_file_data(self, filename):
+    def read_wallet_file_data(self, filename, pwd=None):
```

Comment:
This non-intrusive addition to wallet setup for testing (see TestWallet class in commontest.py) was to allow automatically opening an encrypted wallet. Seems not to be an issue, but of course do review.